sequence_tagging_for_ner/infer.py

```diff
@@ -0,0 +1,62 @@
+import gzip
+
+import reader
+from network_conf import *
+from utils import *
+
+
+def infer(model_path, batch_size, test_data_file, vocab_file, target_file):
+    def _infer_a_batch(inferer, test_data, id_2_word, id_2_label):
+        probs = inferer.infer(input=test_data, field=["id"])
+        assert len(probs) == sum(len(x[0]) for x in test_data)
+
+        for idx, test_sample in enumerate(test_data):
+            start_id = 0
+            pred_str = ""
+            for w, tag in zip(test_sample[0],
+                              probs[start_id:start_id + len(test_sample[0])]):
+                pred_str += "%s[%s] " % (id_2_word[w], id_2_label[tag])
```

Comment:
Since lowercase is used, the outputs might be different from the raw texts. It should be noticed if the raw texts are wanted.